stories/guides/testdatacreator.stories.mdx

```diff
@@ -0,0 +1,40 @@
+<Meta title="Documentation/Guides/Test Data Creator"/>
+
+# Test Data Creator
+
+## What is it?
+
+Some parts of a user journey require data before we can begin automation testing. For example, in order to complete end to end tests for the checkout page, we would first need to go through user authentication (unless guest), choosing a restaurant, a menu and more. This package creates the data for us, enabling us to get to the page we need and automate the tests straight away.
+
+In this package, the environment config and service type are passed in as parameters, and the respective data is returned. It could also be useful for developers, since it can quickly generate data without the need of a browser. The package currently works for QA1 and staging-uk.
```

Comment:
Do we have any concerns in mentioning environment names?

Reply:
Ah yeah good point - I'll remove it just to be safe :)

Reply:
Agreed. I've just add to the section of the bottom with. 'Please contact Web Core' for environment specific configuration files.